easybuild/easyconfigs/b/BAMM/BAMM-2.5.0-foss-2016b.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     GPL-v3.0
+#
+# Notes::
+##
+
+easyblock = 'CMakeMake'
+
+name = 'BAMM'
+version = '2.5.0'
+
+homepage = 'http://bamm-project.org/'
+description = """ BAMM is oriented entirely towards detecting and quantifying heterogeneity in evolutionary rates. 
+It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of 
+lineage diversification and trait evolution. """
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/macroevolution/bamm/archive']
+sources = ['v%(version)s.tar.gz']
+
```

Comment:
For file integrity, MD5 is indeed sufficient.

But if we're going to add checksums everywhere, we may as well go the extra mile and provide SHA256 checksums that also protect you from man-in-the-middle attacks when downloading the sources.

Computing a SHA256 checksum indeed takes a little bit longer, but that should be negligible in the overal build & install process...